sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/KnownWindowsVirtualMachineImage.java

```diff
@@ -4,7 +4,12 @@
 
 /** The popular Azure Windows images. */
 public enum KnownWindowsVirtualMachineImage {
-    /** Windows 10 2020 H1 Pro */
+    WINDOWS_DESKTOP_10_PRO("MicrosoftWindowsDesktop", "Windows-10", "win10-22h2-pro"),
```

Comment:
Probably need to move to the end of enum. Order change of enum constants would be considered breaking change for CI.

Reply:
Also javadocs.

Reply:
Fixed in the new version.